pkg/parser/types.go

```diff
@@ -67,7 +67,9 @@ func NewKind(kind string, namespaced bool, supportedVerbs sets.Set[string], exte
 		namer: namer.Namer{
 			Finalize: util.UpperFirst,
 			Exceptions: map[string]string{
-				"Endpoints": "Endpoints",
+				"Endpoints":               "Endpoints",
```

Comment:
Can we add a code comment. Looking just to code without background its confusing why we need this